packages/api/src/schema/api-schema.yaml

```diff
@@ -176,6 +190,11 @@ components:
         name:
           type: string
           example: test_webhook
+        kind:
+          type: string
+          example: webhook
+        userId:
+          type: string
```

Comment:
As mentioned above, both SDK and API should have exactly the same responses. Since we have `userId` on rest api, we would also need for the SDK as well. If we plan to remove this, we should do the same for the rest api as well.